### PR TITLE
Fix daily runs: Adapt to the new format of Prometheus metrics

### PR DESCRIPTION
**Pull request by fedinskiy**
See https://github.com/quarkusio/quarkus/issues/30343 for details

### Summary

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Review by pjgg — APPROVED** [review]
LGTM

**Comment by rsvoboda** [comment]
Commit message is so vague, this can't go in -  `Commits are well encapsulated and follow the best practices`

c'mon, some context needs to be in the message

otherwise we have just row of "Fix error in daily runs" commits

**Review by pjgg — CHANGES_REQUESTED** [review]
Please update PR and commit message, with a better description.

File: monitoring/micrometer-prometheus/src/test/java/io/quarkus/ts/micrometer/prometheus/HttpServerMetricsIT.java [hunk 1]
```diff
@@ -22,7 +22,13 @@ public class HttpServerMetricsIT {
     private static final int ASSERT_METRICS_TIMEOUT_MINUTES = 1;
     private static final List<String> HTTP_SERVER_REQUESTS_METRICS_SUFFIX = Arrays.asList("count", "sum", "max");
 
-    private static final String HTTP_SERVER_REQUESTS_METRICS_FORMAT = "http_server_requests_seconds_%s{method=\"GET\",outcome=\"SUCCESS\",status=\"200\",uri=\"%s\",}";
+    /*
+     * In versions before 2.16 metrics have format '{a,b,}' (with trailing comma)
```
Comment (pjgg) [inline monitoring/micrometer-prometheus/src/test/java/io/quarkus/ts/micrometer/prometheus/HttpServerMetricsIT.java:26]: :heart:

**Review by pjgg — APPROVED** [review]
LGTM

**Pull request merged**